dataprofiler/tests/profilers/test_base_option.py

```diff
@@ -17,12 +17,21 @@ class TestBaseOption(unittest.TestCase):
 	def setUpClass(cls):
 		cls.data = Data(data=pd.DataFrame([1, 2]), data_type='csv')
 	
+
+	@classmethod
+	def getOptions(self, **params):
+		return BaseOption(**params)
```

Comment:
should name it as `getBaseOption`

Reply:
But shouldn't that not matter if in the boolean tests you just use super().<test_function>? Because then it'll defer to the super's implementation which can be named something different, and it'll still work in the boolean case? Idk maybe I'm misunderstanding the goal.

Reply:
Consider the super().test_validate_helper() call in test_validate_helper for TestBooleanOption. If I used getBaseOption in this class, then regardless of which class is calling super, it will create a new BaseOption object. By creating a getOptions method that is overriden the getOptions call in the TestBaseOption class will create a BooleanOption (when TestBooleanOption is calling its super) which is the intended behavior.

Reply:
It seems I had a lapse in my python understanding, I thought in the call to super() it wouldn't make calls to the Boolean class, I just tested this out in a jupyter notebook and realized I was wrong. Disregard my comment.

Reply:
`setUpClass` is in camel case because of the design of unittest, but i'm not sure `getOptions` should also be camel case but rather snake case as normalized for python.